source/features/pr-merge-info-enhancements.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import debounce from 'debounce-fn';
+import delegate from 'delegate-it';
+import features from '../libs/features';
+
+const inputMap = new WeakMap();
+
+const createCommitTitle = debounce((): string =>
+	`${select('.js-issue-title')!.textContent!.trim()} (${getPRNumber()})`
+, {
+	wait: 1000,
+	immediate: true
+}) as () => string;
+
+function getPRNumber(): string {
+	return select('.gh-header-number')!.textContent!;
+}
+
+// Updates the field and dispatches the right events (focus is also used by `fit-textareas`)
+function updateField(selector: string, value: string): void {
```

Comment:
Why explicit return type of `void`?

If required then why not at mentioned [here](https://github.com/sindresorhus/refined-github/pull/1934/files#diff-7aa792454424677c11856ea64b139ab9R69) too?

Reply:
Explicit return type prevents bugs where you accidentally change the code inside the function to return a different type.

Reply:
But `void` is not used for other functions, in the current feature and other features too.

Either they should be used everywhere, for all functions, or for none (implicit).

Reply:
It will be soon 

https://github.com/sindresorhus/refined-github/pull/1740#discussion_r249408719